index.html

```diff
@@ -248,12 +239,59 @@ <h2>Skills</h2>
             </div>
           </li>
         </ul>
+      </section>
+      <section class="get-in-touch">
+        <h3 class="intro-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+<form action="https://formspree.io/f/mpzblelz" method="post">
+
+<ul>
+
+<li>
+<input type="text" required value="First Name" maxlength="30" name="user_name">
+</li>
+<li>
+  <input required type="email" value="Email address" name="user_email" >
+</li>
+<li>
+  <textarea name="user_message" maxlength="500" required id="message" cols="30" rows="10">Enter text here</textarea>
+</li>
+<li><button type="submit">Get in Touch</button></li>
+</ul>
+
+</form>
 
-        <div class="get-in-touch">
-          <p class="intro-to-form"></p>
-        </div>
       </section>
+     
+    
     </main>
-    <footer></footer>
+    <footer>
+
+
+
+<ul class="social-links">
+  <li>
+    <i class="fa-brands fa-twitter"></i>
+  </li>
+  <li>
+    <i class="fa-brands fa-linkedin-in"></i>
+  </li>
+  <li>
+    <img
+      src="./assets/social-media/medium.svg"
+      alt="Medium social media icon "
+    />
+  </li>
+  <li>
+    <img
+      src="./assets/social-media/github.svg"
+      alt="github social media icon "
+    />
+  </li>
+  <li>
+    <i class="fa-brands fa-angellist"></i>
+  </li>
+</ul>
```

Comment:
- [ ] Kindly try to match the Figma design for the social icon, Please make them with the same `font-size`

| Your design | Figma design |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/159473877-a51320a2-b867-4375-80d5-ad04eefa8fa8.png) | ![image](https://user-images.githubusercontent.com/24703171/159473954-7118a396-b91c-4d7d-8d48-b6c09aa4ec2d.png) |